src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String COMMA = ",";
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    public static final int OPERATION_TYPE_POSITION = 0;
+    public static final int AMOUNT_POSITION = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File readFromFile = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(readFromFile))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                if (line.split(COMMA)[OPERATION_TYPE_POSITION].equals("supply")) {
+                    supply += Integer.parseInt(line.split(",")[AMOUNT_POSITION]);
+                }
+                if (line.split(COMMA)[OPERATION_TYPE_POSITION].equals("buy")) {
+                    buy += Integer.parseInt(line.split(COMMA)[AMOUNT_POSITION]);
+                }
+                line = bufferedReader.readLine();
```

Comment:
what about this? https://github.com/mate-academy/jv-work-with-file/pull/72#discussion_r606960656

Reply:
it is inside while loop

Reply:
or I am blind

Reply:
try to add this line to loop condition field, like `while (your line)`